docs/book/cookbook/application-wide-layout.md

```diff
@@ -0,0 +1,85 @@
+# Set Application-Wide Layout
+
+The format for the view helper `FlashMessenger` can be configured for an entire application and all uses of the helper.
+
+`Laminas\View\Helper\Service\FlashMessengerFactory` checks the application
+configuration and creates the view helper with the given format options.
+
+## Set Format for All Namespaces
+
+Add the following lines to the local or global configuration file, e.g. `config/autoload/global.config.php`:
+
+```php
+return [
+    'view_helper_config' => [
+        'flashmessenger' => [
+            'message_open_format'      => '<p%s>',
+            'message_close_string'     => '</p>',
+            'message_separator_string' => '<br>',
+        ],
+    ],
+];
+```
+
+## Set Formats Individually for Namespaces
+
+INFO: **New Feature**
+Available since version 1.8.0
```

Comment:
Do we need to highlight this as new feature?

Reply:
Yes, because the current documentation is not versioned and if versions are used in the documentation, it is only for major versions.